app/views/mappings/tab_sections/_upload_mappings.html.haml

```diff
@@ -0,0 +1,16 @@
+.upload-mappings
+    .card.upload-mappings-example
+        .title-bar{"data-target" => "#collapseOne", "data-toggle" => "collapse"}
+            = t("mappings.bulk_loader.loader.example_of_valid_file")
+        #collapseOne.collapse
+            .card-body
+            %pre
+                %code
+                = JSON.pretty_generate @example_code
+        
+    = form_with url: '/mappings/loader', method: :post, multipart: true, data: { turbo: true, turbo_frame: 'file_loader_result'} do
+        %div.mb-3
+            = render Input::FileInputComponent.new(name: :file)
+        = render Buttons::RegularButtonComponent.new(id:'upload-mappings-button', value: "Save", variant: "primary", size: 'slim', type:'submit', state: "regular")
```

Comment:
use the existent `form_save_button` helper (and remove `save_button_component` as it is a duplicate)